src/buildercore/cfngen.py

```diff
@@ -171,6 +145,43 @@ def _parameterize(string):
     return context
 
 
+def build_context_rds(context, existing_context):
+    if 'rds' not in context['project']['aws']:
+        return {}
+    stackname = context['stackname']
+
+    # deletion policy
+    # enforce here rather than allow to be specified in project config
+    deletion_policy = 'Delete'
+    if context['is_prod_instance'] or context['instance_id'] in ['continuumtest', 'end2end']:
```

Comment:
tricky to have these keys specified in keys rather than configuration :thinking:

Reply:
I would prefer to go for safety and here have a default of `Snapshot`; in https://github.com/elifesciences/builder/blob/master/projects/elife.yaml#L165-L170 add a `rds` key alongside `ec2` with the override to `Delete`. It lets us treat adhoc environments (I just created `journal--annotationstest` that will be used for a month or so) as akin to production, while formula builds using `standalone` really are throwaway stacks.

Reply:
tweaking the logic so default is Snapshot is easy.

I originally had the deletion policy specified in the projects.yaml file and found myself repeating the line `deletion-policy: Delete` again and again and again. Repeating something that often is bound to cause a mistake. I prefer enforcing this in code, but there is room for an explicit override that need not be repeated. 

But yes - those keys are just magic strings floating there. I can reinforce them conf.py or core.py, but I don't seem to recall builder knowing too much about named environments. i'll investigate

Reply:
I'm grepping through builder's source and there is a distinct lack of code doing environment checking, which I like. I wish our formula's were this free of conditional behaviour. It's going to make the project files even more verbose, but I'll go with your recommendation: default to `Snapshot` and have the alt-configs explicitly set to `Delete`